app/models/concerns/type_of_means_assessment.rb

```diff
@@ -29,6 +29,15 @@ def requires_full_capital?
     ].include?(CaseType.new(kase.case_type))
   end
 
+  def include_partner_in_means_assessment?
```

Comment:
Would it be simpler to default this to false and then only return true if the involvement in case is none or codefendant with no conflict of interest?

Reply:
Possibly. Will try and let you know :)